src/Core.Tests/BinReader.Tests.fs

```diff
@@ -1,4 +1,4 @@
-(*
+﻿(*
```

Comment:
Seems like you inserted a [BOM](https://en.wikipedia.org/wiki/Byte_order_mark).

You can easily remove it by running `dos2unix` on the file.